pkgs/development/libraries/libbluray/default.nix

```diff
@@ -20,7 +20,10 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-wksPQcW3N7u2XFRP5jSVY3p3HBClGd/IAudp8RK0O3U=";
   };
 
-  patches = optional withJava ./BDJ-JARFILE-path.patch;
+  patches = [
+    (optional withJava ./BDJ-JARFILE-path.patch)
+    (optional withJava ./libbluray-1.3.1-Fix-build-failure-after-Oracle-Java-CPU-for-April-2022.patch)
```

Comment:
patches should not be optional and be written in a way that they do not break when the feature is not enabled

Reply:
The BDJ patch was already optional, meanwhile the second patch do not break anything if the feature is not enabled, as that is only related to the compilation of the java feature.

Reply:
Then please apply them both all the time to prevent patch apply failures in the future